mateo-mustapic/assignments/liferay-app/sdk/portlets/todo-portlet/docroot/js/main.js

```diff
@@ -70,7 +155,27 @@ AUI.add(
 							todoList.append(taskHtml);
 
 							taskInput.val('');
+
+							instance._updateTaskCounts();
 						}
+					},
+
+					_updateTaskCounts: function() {
+						var instance = this;
+
+						var todoList = instance._todoList;
+
+						var taskListItems = todoList.all('li');
+
+						var remainingCount = taskListItems.size();
+
+						var remainingDiv = instance._remainingDiv;
+
+						remainingDiv.html(remainingCount);
+
+						var profileTasksCount = instance._profileTasksCount;
+
+						profileTasksCount.html(remainingCount);
```

Comment:
This line throws error in log when you click 'Add'.

Before using nodes that are result of DOM query, always check if it is found:
```
if (profileTasksCount) {
 // do stuff
}
```